### DISCUSSION TITLE
Error while starting the shiny app

### DISCUSSION THREAD

**Discussion by anonymousaurav** [tamnva/SWATshiny#55] [Q&A] [answered]
When I started the Shiny app. The code is showing the following error.

![Screenshot 2023-09-05 130836](https://github.com/tamnva/R-SWAT/assets/29114934/d577586c-c43e-4d79-998d-20c0955d5bb1)

Listening on http://127.0.0.1:5055
Warning: Error in dataTableOutput: unused arguments (width = "100%", height = "1px")
  85: dots_list
  84: div
  83: column
  74: runSwatUI [C:\Users\s.pradhananga\Downloads\R-SWAT-master\R-SWAT-master\R\runSWATUI.R#11]
   1: runApp

What could be causing this issue?
What would be the solution to this issue?

**Comment by tamnva** [comment, marked as answer]
thanks for pointing this out. I have updated the code, it should work now

**Reply by anonymousaurav** [reply]
Thank you, it worked for me.
